frontend/src/static/js/components/webstatus-columns-dialog.ts

```diff
@@ -58,41 +69,77 @@ export class WebstatusColumnsDialog extends LitElement {
 
   handleSave() {
```

Comment:
Could you either:
1. Add a unit test to the existing dialog test [file](https://github.com/GoogleChrome/webstatus.dev/blob/main/frontend/src/static/js/components/test/webstatus-columns-dialog.test.ts) that:
   - Selects the column options check box
   - Add a [spy](https://sinonjs.org/releases/latest/spies/) on formatUrlWithColumns
   - call component.handleSave() in the test
   - have the spy assert that it was called with the right options
     - Do it for:
       - Case 1. no column options saved
       - Case 2. Baseline high date option saved
       - Case 3. Baseline low date option saved
       - Case 4. Both options saved.
     - Even though you fixed the `!cb.checked` already, these cases would help keep that covered in the future.
2. Or, add a playwright test that
   - Opens the page with the default columns
   - Opens the column dialog
   - Selects the column options (similar cases as above). Save
   - Detect the options in the url.
   - Optionally, take a screenshot